### DISCUSSION TITLE
Missing Branch Search when raising a pull request

### DISCUSSION THREAD

**Discussion by voltaire-kahunas** [community/community#129279] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

There used to be a search text field for branches when clicking the target branch names when raising a pull request. Now its missing. Did github intentionally removed it?

![image](https://github.com/community/community/assets/166114966/224ef58d-93d2-4f64-b437-b610202dc259)

**Comment by samrat-rapidops** [comment]
I believe they made a mistake, but what's unacceptable is that I've been facing this issue for the past few hours, and it still hasn't been resolved by an organization like GitHub.

**Reply by DarshanVesatiya** [reply]
Yes, It is becoming a pain in the ***.

**Reply by smp2103** [reply]
깃허브는 각성하라

**Comment by mahdiar-naufal-shyftplan** [comment]
What makes it worse is the list is not sorted

**Comment by dazlee** [comment]
<img width="825" alt="Screen Shot 2024-06-21 at 5 21 40 PM" src="https://github.com/community/community/assets/499822/d6484894-f3e6-409d-a72b-f7d73461f0fe">

<img width="806" alt="Screen Shot 2024-06-21 at 5 24 50 PM" src="https://github.com/community/community/assets/499822/0503b642-19d9-4254-917e-d1ba4ad95b39">


for workaround, just drag SelectMenu-filter and drop upper <tab-container....

**Comment by DarshanVesatiya** [comment]
It is resolved now.
